### PR TITLE
Replace xenvm/xenops calls with xl ones

### PR DESCRIPTION
**Pull request by jean-edouard**
OXT-916

### CODE REVIEW

**Comment by rossphilipson** [comment]
Build here (testing now):
http://openxt.ainfosec.com/builds/custom/master/584

File: recipes-openxt/xenclient/xenclient-dom0-tweaks/dom0_add_vif.sh [hunk 1]
```diff
@@ -19,19 +19,18 @@
...
 while true; do
-        timeout 5 xenops add_vif -backend-domid "$backend" -domid 0 -mac "$local_mac" -devid 0
+        xec -s com.citrix.xenclient.networkdaemon add-vif 0 $backend $local_mac
         [ "$?" -eq 0 ] && break;
-        xenops del_vif -backend-domid "$backend" -domid 0 -mac "$local_mac" -devid 0
+	sleep 1
 done
 
-ifconfig eth0 inet 0.0.0.0
-if [ ! -e /config/system/dom0-networking-disabled ]; then
```
Comment (rossphilipson) [inline recipes-openxt/xenclient/xenclient-dom0-tweaks/dom0_add_vif.sh]: Why was this check removed? It looks like the toolstack still writes this file out.

Comment (jean-edouard) [inline recipes-openxt/xenclient/xenclient-dom0-tweaks/dom0_add_vif.sh]: It was moved to the top of the script.
No need to do anything here if that flag is set.

**Comment by rossphilipson** [comment]
Tested this and the other 4 related PRs. I don't see any problems with the removal of xenvm/xenops. Merging these soon.

**Pull request merged**